pangeo-notebook/environment.yml

```diff
@@ -48,6 +52,14 @@ dependencies:
  - sparse
  - tiledb-py
  - xarray
+ - xcape
  - xesmf
+ - xgcm
+ - xhistogram
+ - xlayers
+ - xmitgcm
  - xpublish
+ - xrft
  - zarr
+ - pip:
+   - fastjmd95
```

Comment:
As soon as https://github.com/conda-forge/staged-recipes/pull/11980 is merged, we can install this from conda.